weblate/vendasta/auth.py

```diff
@@ -15,14 +15,15 @@ class VendastaOpenIdConnect(OpenIdConnectAuth):
 
     @cache(ttl=86400)
     def oidc_config(self):
-        oidc_endpoint = self.setting("OIDC_ENDPOINT", "http://iam-prod.vendasta-internal.com")
+        oidc_endpoint = self.setting(
+            "OIDC_ENDPOINT", "http://iam-prod.vendasta-internal.com"
```

Comment:
this is ok to always be pointed at prod?

Reply:
The second parameter is a fallback for if the "OIDC_ENDPOINT" configuration is not found.

Reply:
ah, gotcha! 👍